src/main/java/com/iexec/worker/compute/ComputeStageConverter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.worker.compute;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ComputeStageConverter implements Converter<String, ComputeStage> {
+    @Override
+    public ComputeStage convert(String value) {
+        return ComputeStage.valueOf(value.toUpperCase());
```

Comment:
Can you specify where it is exactly used?

Reply:
Spring loads this component and uses it when receiving a request on `ComputeController#sendExitCauseForGivenComputeStage`.

Reply:
Could we just add what you just explained as a javadoc comment in this conversion class ?

Reply:
Sure, see https://github.com/iExecBlockchainComputing/iexec-worker/pull/454/commits/fafaf73d68996c1d56f8675bfba3590997159c7a.

Reply:
Thanks!